src/fights.ts

```diff
@@ -1474,6 +1479,13 @@ const freeRunFightSources = [
   ),
 ];
 
+function sandwormRequirement() {
+  return new Requirement(
+    ["100 Item Drop"],
+    get("garbageChampagneCharge") > 0 ? { forceEquip: $items`broken champagne bottle` } : {}
```

Comment:
does anything verify that you own the champagne bottle?

Reply:
done